.gitlab-ci.yml

```diff
@@ -573,8 +565,7 @@ API-module-hotfixes:
   parallel:
     matrix:
       - RUNNER:
-          - rhos-01/fedora-38-x86_64
-          # - rhos-01/fedora-39-x86_64
```

Comment:
Maybe we should add comment here why 39 is skipped (if this is the same case as in line 483?)

Reply:
there's no openstack image for fedora39 currently